packages/edit-post/src/components/header/style.scss

```diff
@@ -1,19 +1,22 @@
 .edit-post-header {
-	height: $header-height;
 	padding: $grid-size-small 2px;
 	border-bottom: $border-width solid $light-gray-500;
 	background: $white;
 	display: flex;
-	flex-direction: row;
-	align-items: stretch;
+	flex-flow: row wrap;
 	justify-content: space-between;
+	max-width: 100vw;
```

Comment:
We should always be careful when using the vw unit, because it does not account for the presence of visible scrollbars. So using 100vw might unintentionally cause a horizontal scrollbar.

Reply:
Good point. I checked Chrome and Safari on Mac, Chrome and IE on Windows 7 and Windows 10, and it's not happening, though likely that's because we have `overflow-x: hidden` on the body element for screens under 782px. If we removed that, we'd get a massive horizontal scroll because of the admin bar having a min-width of 300px. That, and a couple other min-widths set on the core side, are the reason I had to resort to `vw` in the first place, to get the top bar to not overflow the viewport.
I'll check trac to see if there's a ticket for addressing that, and create one if there isn't, but suspect it will be non-trivial to fix on that side 😅so for now we're probably better off with the `vw` solution.